itest/godot/ManualFfiTests.gd

```diff
@@ -68,6 +68,35 @@ func test_export():
 	obj.free()
 	node.free()
 
+func test_export_dyn_gd():
+	var dyn_gd_exporter = RefcDynGdExporter.new()
+
+	# NodeHealth is valid candidate both for `empty` and `second` fields.
+	var node = NodeHealth.new()
+	dyn_gd_exporter.first = node
+	assert_eq(dyn_gd_exporter.first, node)
+
+	dyn_gd_exporter.second = node
+	assert_eq(dyn_gd_exporter.second, node)
+
+	# RefcHealth is valid candidate for `first` field.
+	var refc = RefcHealth.new()
+	dyn_gd_exporter.first = refc
+	assert_eq(dyn_gd_exporter.first, refc)
+	node.free()
+
+func test_export_dyn_gd_should_fail_for_wrong_type():
+	if runs_release():
+		return
+	var dyn_gd_exporter = RefcDynGdExporter.new()
```

Comment:
Empty line, otherwise good
```suggestion
func test_export_dyn_gd_should_fail_for_wrong_type():
	if runs_release():
		return

	var dyn_gd_exporter = RefcDynGdExporter.new()
```